airflow/worker/airflow_worker.py

```diff
@@ -0,0 +1,138 @@
+import asyncio
+import logging
+import os
+from datetime import datetime
+from typing import Any
+
+from airflow import settings
+from airflow.exceptions import AirflowException
+from airflow.models import DAG
+from airflow.models import (
+    DagBag, TaskInstance
+)
+from airflow.utils.log.logging_mixin import (LoggingMixin)
+from airflow.utils.net import get_hostname
+from airflow.utils.state import State
+from concurrent.futures import ProcessPoolExecutor
+
+import pendulum
+
+app = None  # type: Any
+loop: asyncio.AbstractEventLoop = None
+pool = None
+executor = None
+DAGS_FOLDER = settings.DAGS_FOLDER
+
+import asyncio
+from aiohttp import web
+
+
+async def health(request):
+    name = request.rel_url.query["name"]
+    return web.Response(text="Hello, {}".format(name))
+
+
+def myconverter(o):
+    if isinstance(o, datetime):
+        return o.__str__()
+
+
+async def run_task(request):
+    dag_id = request.rel_url.query['dag_id']
+    task_id = request.rel_url.query['task_id']
+    # subdir = request.rel_url.query['subdir']
+    subdir = "/Users/dimberman/airflow/dags"
+
+    execution_date = pendulum.fromtimestamp(int(request.rel_url.query["execution_date"]))
+    log = LoggingMixin().log
+    #
+    log.info("running dag {} for task {} on date {} in subdir {}".format(dag_id, task_id, execution_date, subdir))
+    logging.shutdown()
```

Comment:
What effect does having this here have on the logs from the task? Does it get re-initiallized in the task runner?